.travis/smoke-test.sh

```diff
@@ -34,6 +34,7 @@ echo "Smoke testing api-gateway (scholarly-articles content-store)"
 
 echo "Smoke testing api-gateway (search)"
 [[ "$(curl -sS "http://localhost:${HTTP_PORT_GATEWAY}/search/ping" 2>&1)" == "pong" ]]
+[[ "$(curl -sS "http://localhost:${HTTP_PORT_GATEWAY}/search" --output /dev/null --write-out '%{http_code}' 2>&1)" == "200" ]]
```

Comment:
is the 200 from a redirect to `/items` or is it directly answered by `/`?

Reply:
Directly.